frontend/src/routes/Credentials/CredentialsPage.tsx

```diff
@@ -231,11 +231,7 @@ export function CredentialsTable(props: {
             sort: 'metadata.creationTimestamp',
             cell: (resource) => (
               <span style={{ whiteSpace: 'nowrap' }}>
-                {resource.metadata?.creationTimestamp ? (
-                  <AcmTimestamp timestamp={resource.metadata.creationTimestamp} />
-                ) : (
-                  '—'
-                )}
+                <AcmTimestamp timestamp={resource.metadata?.creationTimestamp ?? ''} />
```

Comment:
```suggestion
                <AcmTimestamp timestamp={resource.metadata?.creationTimestamp} />
```
No need for the nullish coalescing operator here... timestamp is permitted to be undefined now.